CorsixTH/Lua/dialogs/resizables/hotkey_assign.lua

```diff
@@ -434,297 +410,187 @@ function UIHotkeyAssign:UIHotkeyAssign(ui, mode)
     hotkeys_backedUp = true
   end
 
+  self.key_pane = nil
+  self.key_options = {
+    {
+      id = "global",
+      title = _S.hotkey_window.panel_globalKeys,
+      tooltip = _S.hotkey_window.panel_globalKeys,
+      sections = {{
+          title = _S.hotkey_window.panel_globalKeys,
+          keys = {
+            "global_exitApp",
+            "global_resetApp",
+            "global_screenshot",
+            "global_releaseMouse",
+            "global_confirm_alt",
+            "global_cancel_alt" }}}},
+    {
+      id = "ingame",
+      title = _S.hotkey_window.panel_generalInGameKeys,
+      tooltip = _S.hotkey_window.panel_generalInGameKeys,
+      sections = {{
+          title = _S.hotkey_window.panel_generalInGameKeys,
+          keys = {
+            "ingame_showmenubar",
+            "ingame_saveMenu",
+            "ingame_loadMenu",
+            "ingame_jukebox",
+            "ingame_openFirstMessage",
+            "ingame_quickSave",
+            "ingame_quickLoad",
+            "ingame_restartLevel",
+            "ingame_quitLevel" }}}},
+    {
+      id = "speed",
+      title = _S.hotkey_window.button_gameSpeedKeys,
+      tooltip = _S.tooltip.hotkey_window.button_gameSpeedKeys,
+      sections = {{
+          title = _S.hotkey_window.panel_gameSpeedKeys,
+          keys = {
+            "ingame_pause",
+            "ingame_gamespeed_slowest",
+            "ingame_gamespeed_slower",
+            "ingame_gamespeed_normal",
+            "ingame_gamespeed_max",
+            "ingame_gamespeed_thensome",
+            "ingame_gamespeed_speedup" }}}},
+    {
+      id = "zoom",
+      title = _S.hotkey_window.panel_zoomKeys,
+      tooltip = _S.hotkey_window.panel_generalInGameKeys,
+      sections = {{
+          title = _S.hotkey_window.panel_zoomKeys,
+          keys = {
+            "ingame_zoom_in",
+            "ingame_zoom_in_more",
+            "ingame_zoom_out",
+            "ingame_zoom_out_more",
+            "ingame_reset_zoom" }}}},
+    {
+      id = "misc",
+      title = _S.hotkey_window.panel_miscInGameKeys,
+      tooltip = _S.hotkey_window.panel_miscInGameKeys,
+      sections = {{
+          title = _S.hotkey_window.panel_miscInGameKeys,
+          keys = {
+            "ingame_rotateobject",
+            "ingame_patient_gohome",
+            "ingame_setTransparent" }}}},
```

Comment:
imo this should be directly below "General In-Game Keys" for consistency.

Reply:
Convince me? I was thinking it might belong at the bottom or between Recall and Debug (all of the keys except Global are in-Game - but I didn't make up the categories)

Reply:
It could also be at the bottom sure.
The reason behind that is game keys and misc game keys are going to be at least somewhat related. Putting them next to each other would make more sense .

However, if you're treating them like a general miscellaneous category put them at the bottom.